package.json

```diff
@@ -68,5 +68,8 @@
     "stack-trace": "0.0.10",
     "uuid": "^9.0.1"
   },
-  "keywords": []
+  "keywords": [
+    "error",
+    "reporting"
```

Comment:
Some more ideas (they don't have to be single words, I think)

crash reporting
error tracking
raygun
APM

Reply:
seems to be a mix of different formats: https://www.npmjs.com/search?q=crash

e.g. `crash` `reporting` `crash-reporting` `crash reporting`

Reply:
If you search for "crash reporting" you already get results with packages with them as separated words. I'd stick to that.

![image](https://github.com/MindscapeHQ/raygun4node/assets/2494376/e5438cf7-9e50-4fe6-9dab-6549011bf19a)

Reply:
debug
logging

might be relevant as well.